src/uct/ib/rdmacm/rdmacm_cm_ep.c

```diff
@@ -28,9 +32,23 @@ const char* uct_rdmacm_cm_ep_str(uct_rdmacm_cm_ep_t *cep, char *str,
         NULL
     };
 
+    if (ucs_sockaddr_is_known_af(local_addr)) {
+        ucs_sockaddr_str(local_addr, local_ip_port_str, UCS_SOCKADDR_STRING_LEN);
+    } else {
+        ucs_strncpy_zero(local_ip_port_str, "<not set>", UCS_SOCKADDR_STRING_LEN);
+    }
+
+    if (ucs_sockaddr_is_known_af(remote_addr)) {
+        ucs_sockaddr_str(remote_addr, remote_ip_port_str, UCS_SOCKADDR_STRING_LEN);
+    } else {
+        ucs_strncpy_zero(remote_ip_port_str, "<not set>", UCS_SOCKADDR_STRING_LEN);
+    }
+
     ucs_flags_str(flags_buf, sizeof(flags_buf), cep->flags, ep_flag_to_str);
-    ucs_snprintf_safe(str, max_len, "rdmacm_ep %p, status %s, flags %s",
-                      cep, ucs_status_string(cep->status), flags_buf);
+    ucs_snprintf_safe(str, max_len, "rdmacm_ep %p local address %s, "
+                      "remote address %s ep_status %s flags %s",
+                      cep, local_ip_port_str, remote_ip_port_str,
+                      ucs_status_string(cep->status), flags_buf);
```

Comment:
need to make it shorter, it's printed for every message
`"[%p %s->%s %s %s]"`
`cep, local_address, remote_address, flags_buf, status == OK ? "" : ucs_status_string(status)`
and remove `:` form all places uct_rdmacm_cm_ep_str() is used like this "%s: ..."

Reply:
what do you mean "for every message"? it's printed only in case of an error or for logging from rdmacm api calls...
but ok, i'll just keep cep ok? so that it would be clear for the reader that doesn't know the context of the print:
"[**cep** %p %s->%s %s %s]"

Reply:
yes
i meant that for many rdmacm log/errors we use cep_str, so it has to be as short as possible to keep log lines within reasonable length

Reply:
also, let's print UCS_OK if the status is ok... the reader won't know the status otherwise (unless he looks into the code)